Python/Stream/pyEncryptedStream.cpp

```diff
@@ -112,7 +112,12 @@ PY_METHOD_STATIC_VA(EncryptedStream, IsFileEncrypted,
         PyErr_SetString(PyExc_TypeError, "IsFileEncrypted expects a string or an os.PathLike object");
         return nullptr;
     }
-    return pyPlasma_convert(plEncryptedStream::IsFileEncrypted(filename));
+    try {
+        return pyPlasma_convert(plEncryptedStream::IsFileEncrypted(filename));
+    } catch (...) {
+        PyErr_SetString(PyExc_IOError, "Error opening file");
```

Comment:
If you catch `const std::exception&`, you can get the actual failure from libhsplasma from `std::exception::what()`.

Reply:
I copy pasta'd this from all the other error handlers in the file ^_^